web/src/composables/useLogs.ts

```
@@ -3826,7 +3834,10 @@ const useLogs = () => {
     }
 
     searchObj.shouldIgnoreWatcher = false;
-    if(queryParams.hasOwnProperty("type") &&  queryParams.type == "search_history_re_apply"){
+    if (
+      queryParams.hasOwnProperty("type") &&
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)